Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsHomeCard.kt

```diff
@@ -51,6 +53,23 @@ class StatisticsHomeCard(
                         verticalPadding = R.dimen.spacing_tiny
                     )
                 )
+                addOnScrollListener(object : RecyclerView.OnScrollListener() {
+                    override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
+                        super.onScrolled(recyclerView, dx, dy)
+                        recyclerView.postDelayed(
+                            {
+                                val visibleItem = statisticsLayoutManager.findFirstCompletelyVisibleItemPosition()
+                                val viewHolder = recyclerView.findViewHolderForAdapterPosition(visibleItem)
+                                viewHolder?.itemView?.requestFocus()
+                                viewHolder?.itemView?.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_FOCUSED)
+                                viewHolder?.itemView?.announceForAccessibility(
+                                    "$visibleItem of ${statisticsCardAdapter.itemCount}"
+                                )
+                            },
+                            1000
+                        )
+                    }
+                })
```

Comment:
these another KTX for postDelayed , could you please use it , it is more Kotlin friendly 
```kotlin
recyclerView.postDelayed(1_000L) {
 // Code 
}
```

Reply:
I've removed `postDelayed` as I've realised it wasn't actually needed at all.